### PR TITLE
accept few attributes(variables) in one path

### PR DESCRIPTION
**Pull request by ajanczak**

### CODE REVIEW

File: postman-generator/src/test/resources/apibuilder/test-service-1.json [hunk 1]
```diff
@@ -6,14 +6,14 @@
...
   "application": {
-    "key": "apidoc-reference-api"
-  },
-  "namespace": "io.ecorp.api.v0",
-  "version": "0.3.47",
-  "info": {},
-  "name": "catalog",
-  "base_url": "https://api.ecorp.io",
-  "headers": [ ],
+  "key": "apidoc-reference-api"
```
Comment (anadoba) [inline postman-generator/src/test/resources/apibuilder/test-service-1.json]: formatting is broken here. you can reformat the whole file

File: postman-generator/src/test/resources/apibuilder/test-service-1.json [hunk 2]
```diff
@@ -173,7 +198,63 @@
...
+            "operation_method": "POST",
+            "operation_path": "/:organization/catalog",
+            "identifier_field": "id"
+          }
+        },
+        {
+          "name": "object-reference",
+          "value": {
+            "related_service_namespace": "io.ecorp.api.v0",
+            "resource_type": "catalog",
```
Comment (anadoba) [inline postman-generator/src/test/resources/apibuilder/test-service-1.json:221]: is this `resource_type` correct? 
i think that `subcatalog` is the right one

File: postman-generator/src/main/scala/generator/DependantOperationResolver.scala [hunk 3]
```diff
@@ -39,15 +39,16 @@ object DependantOperationResolver extends Logging {
...
+        path <- resource.path.toSeq
+        parameters = findParametersInPathString(path)
+        foundedPathAttrIdx <- resource.attributes.zipWithIndex if (
+          foundedPathAttrIdx._1.name.equalsIgnoreCase(AttributeName.ObjectReference.toString)
+          && parameters.size > 0)
+        pureAttr <- tryAttributeReadsWithLogging[ObjectReference](foundedPathAttrIdx._1.value).toSeq
         extendedObjectReference = pureAttr.toExtended
-        parameterFromResourcePath = resource.path.flatMap(findParameterInPathString)
-        postmanVariableNameOpt = parameterFromResourcePath.map(postmanVariableNameFrom)
+        postmanVariableNameOpt = postmanVariableNameFrom(parameters(foundedPathAttrIdx._2))
```
Comment (anadoba) [inline postman-generator/src/main/scala/generator/DependantOperationResolver.scala]: looks like `postmanVariableNameOpt` is no longer an `Opt`

File: postman-generator/src/test/resources/apibuilder/test-service-1.json [hunk 4]
```diff
@@ -143,6 +143,31 @@
             }
           ]
         },
+        {
+          "method": "POST",
+          "description": "Create catalog.",
```
Comment (anadoba) [inline postman-generator/src/test/resources/apibuilder/test-service-1.json]: the description is wrong here. It's `subcatalog`

File: postman-generator/src/test/resources/apibuilder/test-service-1.json [hunk 5]
```diff
@@ -143,6 +143,31 @@
...
+          "path": "/:organization/subcatalog",
```
Comment (anadoba) [inline postman-generator/src/test/resources/apibuilder/test-service-1.json:149]: actually, this operation `POST /:organization/subcatalog`
should be moved to `subcatalog` resource  (you've created it in this PR, it already contains GET subcatalog operation)

File: postman-generator/src/test/resources/postman/expected-1.json [hunk 6]
```diff
@@ -176,7 +320,7 @@
                     "content": "Type: uuid  | Required: true"
                   },
                   "disabled": false,
-                  "value": "{{organization}}",
+                  "value": "{{catalogs-organization}}",
```
Comment (anadoba) [inline postman-generator/src/test/resources/postman/expected-1.json]: this variable name looks strange.
I'd say that `organization` should be here.

Comment (anadoba) [reply postman-generator/src/test/resources/postman/expected-1.json]: maybe it's because of heuristics?

**Review by anadoba — CHANGES_REQUESTED** [review]
2 things should be changed before merging 
- test service definition rearrangements (as in my comments below), as it's the most important test case, so its source data needs to be correct
- variable name clash in the output test postman collection (heuristics conflict?)

File: postman-generator/src/test/resources/apibuilder/test-service-1.json (hunk 2 above)
Comment (ajanczak) [reply postman-generator/src/test/resources/apibuilder/test-service-1.json:221]: no, it's correct. It's create subcatalog in catalog

**Review by anadoba — APPROVED** [review]

**Pull request closed without merging**